client-lib/src/main/java/com/ibm/ws/repository/transport/model/RequireFeatureWithTolerates.java

```diff
@@ -0,0 +1,83 @@
+/*******************************************************************************
+* Copyright (c) 2017 IBM Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+package com.ibm.ws.repository.transport.model;
+
+import java.util.Collection;
+import java.util.HashSet;
+
+public class RequireFeatureWithTolerates {
+
+    /** {@inheritDoc} */
+    @Override
+    public int hashCode() {
```

Comment:
As this is essentially a data class, can we put the `hashCode` and `equals` methods at the bottom of the class body and put the fields at the top?

Reply:
Ok